lib/BackgroundJob/ExpireSignalingMessage.php

```diff
@@ -43,11 +44,13 @@ public function __construct(ITimeFactory $timeFactory,
 
 		// Every 5 minutes
 		$this->setInterval(60 * 5);
+		$this->setTimeSensitivity(IJob::TIME_INSENSITIVE);
```

Comment:
Not a real usecase as I think instances that need the maintenance feature also need an HPB, but:
```suggestion
		$this->setTimeSensitivity(IJob::TIME_SENSITIVE);
```